LICENSE

```diff
@@ -1,21 +0,0 @@
-MIT License
```

Comment:
Please do not remove the license file itself.

Reply:
Sorry for misunderstood. Restored.